include/EASTL/internal/config.h

```diff
@@ -144,18 +144,21 @@
 //
 #if !defined(EA_CPP14_CONSTEXPR)
 
-	#if defined(EA_COMPILER_MSVC_2015)
-		#define EA_CPP14_CONSTEXPR  // not supported
-		#define EA_NO_CPP14_CONSTEXPR 
-	#elif defined(__GNUC__) && (EA_COMPILER_VERSION < 9000)   // Before GCC 9.0
-		#define EA_CPP14_CONSTEXPR  // not supported
-		#define EA_NO_CPP14_CONSTEXPR 
-	#elif defined(EA_COMPILER_CPP14_ENABLED)
-		#define EA_CPP14_CONSTEXPR constexpr
-	#else
-		#define EA_CPP14_CONSTEXPR  // not supported
-		#define EA_NO_CPP14_CONSTEXPR 
-	#endif
+        #if defined(EA_COMPILER_MSVC_2015)
+                #define EA_CPP14_CONSTEXPR // not supported
+                #define EA_NO_CPP14_CONSTEXPR
+        #elif defined(EA_COMPILER_GNUC) && (EA_COMPILER_VERSION < 9000)         // Before GCC 9.0
+                #define EA_CPP14_CONSTEXPR                                      // not supported
+                #define EA_NO_CPP14_CONSTEXPR
+        #elif defined(EA_COMPILER_CLANG) && (EA_COMPILER_VERSION < 500)         // Before clang5
+                #define EA_CPP14_CONSTEXPR                                      // not supported
+                #define EA_NO_CPP14_CONSTEXPR
+        #elif defined(EA_COMPILER_CPP14_ENABLED)
+        #define EA_CPP14_CONSTEXPR constexpr
```

Comment:
Please indent the macro defines.

Reply:
Fixed. Poor clang-format had broken the indent.